Create introduction.md

willwm opened the pull request:
http://campus.codeschool.com/courses/mastering-github/level/1/section/2/forking-and-cloning

willwm committed:
Create introduction.md

http://campus.codeschool.com/courses/mastering-github/level/1/section/2/forking-and-cloning

codeschool-kiddo commented on introduction.md:
```diff
@@ -0,0 +1,3 @@
+# CodeSchool: Mastering GitHub
```
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.

willwm committed:
Update introduction.md

http://campus.codeschool.com/courses/mastering-github/level/1/section/3/pull-request

willwm committed:
Update introduction.md

willwm committed:
Update introduction.md

codeschool-kiddo committed:
Added new rule about coffee

willwm committed:
Update dojo_rules.md

willwm committed:
Merge pull request #2 from willwm/new_rules

Too many people are spilling coffee

Commit:
Merge remote-tracking branch 'origin/deadly_skills'
